command/lock.go

```diff
@@ -37,54 +35,42 @@ const (
 // LockCommand is a Command implementation that is used to setup
 // a "lock" which manages lock acquisition and invokes a sub-process
 type LockCommand struct {
+	Meta
+
 	ShutdownCh <-chan struct{}
-	Ui         cli.Ui
 
 	child     *os.Process
 	childLock sync.Mutex
-	verbose   bool
+
+	limit        int
+	monitorRetry int
+	name         string
+	passStdin    bool
+	timeout      time.Duration
+	verbose      bool
 }
 
 func (c *LockCommand) Help() string {
 	helpText := `
```

Comment:
Did you consider putting the intro text into the meta object as well and then you'd just return `c.Meta.Help()` here? If we had markdown in here we could strip it for the CLI and generate the web versions with some magic :-)